.dockerignore

```diff
@@ -1,5 +1,16 @@
 *
-!gitea
```

Comment:
At startup docker "clone" local folder to build folder this limit what is copied into workspace but we need